createAccountCore/build.gradle

```diff
@@ -51,6 +51,7 @@ dependencies {
 
     implementation 'androidx.core:core-ktx:1.9.0'
     implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.5.1'
+    implementation 'androidx.activity:activity-compose:1.7.0-beta01'
```

Comment:
You have a compose ui constant, why use a different version?

Reply:
because the ui constant isn't a high enough version

Reply:
atleast not for the androidx.activity:activity-compose
for the other things its fine.

Reply:
I have now realized this is because the main :app module is taking in compose core as a dependency which has the correct version it needs.

Reply:
ok now I'm thoroughly confused because I tried to remove the implementations that the compose core is bringing in from loginCompose, it started having an aneurysm, why isn't app having an aneurysm.